test/helpers.py

```diff
@@ -89,3 +112,166 @@ def _wait_until_query_success(
             f"Query ID: {sfqid}"
             f"Final query status: {status}"
         )
+
+
+def create_pyarrow_iterator(input_data, use_table_iterator):
+    # create nanoarrow based iterator
+    return (
+        NanoarrowPyArrowRowIterator(
+            None,
+            input_data,
+            ArrowConverterContext(
+                session_parameters={"TIMEZONE": "America/Los_Angeles"}
+            ),
+            False,
+            False,
+            False,
+        )
+        if not use_table_iterator
+        else NanoarrowPyArrowTableIterator(
+            None,
+            input_data,
+            ArrowConverterContext(
+                session_parameters={"TIMEZONE": "America/Los_Angeles"}
+            ),
+            False,
+            False,
+            False,
+        )
+    )
+
+
+def create_old_pyarrow_iterator(input_data, use_table_iterator):
```

Comment:
We could just name it `create_vendored_pyarrow_iterator` so we don't need the comment? Similar to `create_pyarrow_iterator`

Reply:
yep, I will update this

Reply:
resolved in #1758